api/openapi/v1/system-agent.yaml

```diff
@@ -64,6 +95,15 @@ paths:
               $ref: '#/components/schemas/operation'
         required: true
       responses:
+        200:
+          description: an array of elements where each element contains
```

Comment:
This is too much detail. Maybe something like `A list of services and their executors.`? We don't need to mention that these executors were requested; that's implied.